frameworks/g-cloud-11/questions/services/serviceDefinitionDocumentURL.yml

```diff
@@ -5,7 +5,7 @@ hint: >
   This document will not be indexed by search on the Digital Marketplace. Use an Open Document Format (ODF) or PDF/A
   (eg .pdf, .odt). (Maximum file size 5MB.)
 
-optional: true
+optional: false
```

Comment:
We can just remove the entire `optional` key, I think - as per the `termsAndConditionsDocumentURL` question

Reply:
We can also get rid of the `empty_message` key below.

Reply:
Done 👍